designsystem/src/main/java/com/babylon/wallet/android/designsystem/composable/MnemonicWordTextField.kt

```diff
@@ -80,17 +90,19 @@ fun MnemonicWordTextField(
             else -> RadixTheme.colors.gray1
         }
         val borderColor = when {
-            enabled && highlightField -> RadixTheme.colors.green1
+            enabled && usedInSeedPhraseVerification -> RadixTheme.colors.gray1
```

Comment:
I think `highlightField` was a better name since we know already that this is `MnemonicWordTextField`.